odoo/addons/base/models/ir_cron.py

```diff
@@ -530,3 +602,20 @@ class ir_cron_trigger(models.Model):
     @api.autovacuum
     def _gc_cron_triggers(self):
         self.search([('call_at', '<', datetime.now() + relativedelta(weeks=-1))]).unlink()
+
+
+class ir_cron_progress(models.Model):
+    _name = 'ir.cron.progress'
+    _description = 'Actions Progress'
+    _rec_name = 'cron_id'
+
+    cron_id = fields.Many2one("ir.cron", index=True)
+    remaining_tasks = fields.Integer()
+    progress = fields.Integer()
```

Comment:
To me `progress` suggests some percentage.  In your case, "progress" means "number of processed items", and `remaining_tasks` means "number of remaining items", but not quite so, since in practice it is maxed by `batch_size+1`.

Reply:
Does `done` & `remaining` feel better ? 
I hesitated with `done_tasks` and `remaining_tasks` but they feel a mouthful